aexpect/ops.py

```diff
@@ -473,3 +477,94 @@ def extract_tarball(session, tarball, target_dir, flags="-ap"):
         output = output.strip()
     if status != 0:
         raise RuntimeError(f'Failed to extract {tarball} to {target_dir}: {output}')
+
+
+###############################################################################
+# system
+###############################################################################
+
+
+def start_services(session, services, timeout=60):
+    """
+    Try starting all services in given list.
+
+    :param session: virtual machine session
+    :type session: :py:class:`aexpect.ShellSession`
+    :param services: list of services to modify
+    :type services: [str]
+    :param int timeout: timeout for each service to start
+    :returns: statuses for each modified service
+    :rtype: [int]
+    """
+    statuses = []
+    for service in services:
+        status = session.cmd_status(f'service {service} status')
+        if status == 0:      # is running --> skip
+            LOG.debug(f'Service {service} already started - skipping')
```

Comment:
logs should use lazy prints rather than f-strings.

Reply:
See https://github.com/avocado-framework/aexpect/pull/106#issuecomment-1301842767 but I guess you insist on this